R/summarise_scores.R

```diff
@@ -42,12 +39,12 @@
 #' summarise_scores(scores, by = c("model", "target_type"))
 #'
 #' # get standard deviation
-#' summarise_scores(scores, by = "model", FUN = sd)
+#' summarise_scores(scores, by = "model", func = sd)
```

Comment:
Is `func` better than `FUN`? `FUN` seems quite standard for many functions

Reply:
Can we at least have `fun`? :eyes:

Reply:
Can have FUN but outside of base having arguments with capitals is not really standard. Can of course have fun or whatever else you want. `func` was just a random suggestion.

Reply:
`purrr::map` uses `.f` which could work?

Reply:
(my main concern here is avoiding linting errors as too lazy to manually fix them 😆 )

Reply:
(so FUN is totally fine if you really want as well)